.github/config/labeler-config.yml

```diff
@@ -11,4 +11,9 @@ jdbc:
   - client/trino-jdbc/**
 
 docs:
-  - docs/**
+  # any change under docs except release.rst and release directory
+  - any: ['docs/**', '!docs/src/main/sphinx/release/**', '!docs/src/main/sphinx/release.rst']
```

Comment:
This are still `docs` changes. Let's keep as it was.

Reply:
done.